### PR TITLE
Add side note for a change of default proxy port

### PR DESCRIPTION
**Pull request by imWildCat**
discussion: https://discord.com/channels/1084848369073131531/1084848372667654248/1289001901504860211

### CODE REVIEW

**Comment by imWildCat** [comment]
going out for a little bit will check the PR feedbacks after my dinner (MDT)

File: docs/upgrading/configuration-changes.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ proxy:
     timeout: 2
 ```
 
+Please note the default port for healthcheck is changed from `3000` to `80`. You might need to change this port back to `3000` if you are upgrading from v1.
```
Comment (djmb) [inline docs/upgrading/configuration-changes.md:83]: That is the port the proxy uses for all requests, so it's not healthcheck specific. Maybe we could add an example and word it something like this:

Please note that the healthchecks will use the `app_port` setting which defaults to port 80. Previously healthchecks defaulted to port 3000. You can change this back with:

```
proxy:
  app_port: 3000
```

Comment (imWildCat) [reply docs/upgrading/configuration-changes.md]: thanks! make sense!

updated.

**Comment by djmb** [comment]
A few people have hit this, so thanks for fixing it!

**Comment by imWildCat** [comment]
> A few people have hit this, so thanks for fixing it!

no problem at all!

we all appreciate your team for creating this amazing tool!

**Pull request merged**